src/generic_core/remote-connection.ts

```diff
@@ -198,6 +198,7 @@ import uproxy_core_api = require('../interfaces/uproxy_core_api');
           },
           false
       ).then((endpoint :net.Endpoint) => {
+        log.info('SOCKS proxy listening on %1', endpoint);
```

Comment:
I think it's harder and won't work if we bind to `::1` (if/when we do, we'll surely forget to update this logging statement -- if we don't, it's one more thing to track).